ui/src/errorhandler.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button } from "@mui/material";
```

Comment:
This file should be be "errorPage.tsx" to match the component and standard casing.

Reply:
renamed